src/components/Form/CreateEventForm.jsx

```diff
@@ -164,15 +161,14 @@ function CreateEventForm() {
           url: "https://api.pinata.cloud/pinning/pinFileToIPFS",
```

Comment:
[Boulder] Make sure to turn this into a constant/environment variable that you can read using dotenv